ReadMe.org

```diff
@@ -13,3 +13,5 @@
        python manage.py runserver  
      #+END_SRC sh
 
+** License
+   - GNU GPL v3
```

Comment:
@nutannivate Can you please add GNU GPL v3 image? something like,

![gnu-gpl-image](https://user-images.githubusercontent.com/909861/41195123-5d459694-6c45-11e8-9c61-cf0fbc9bcf1f.png)